homeassistant/components/venstar/sensor.py

```diff
@@ -132,4 +202,20 @@ def native_value(self) -> float:
         cls=VenstarTemperatureSensor,
         stype="temp",
     ),
+    VenstarSensorEntityDescription(
+        key="battery",
+        device_class=DEVICE_CLASS_BATTERY,
+        state_class=STATE_CLASS_MEASUREMENT,
+        native_unit_of_measurement=PERCENTAGE,
+        cls=VenstarBatterySensor,
+        stype="battery",
```

Comment:
Please remove stype and use the key instead, unless there's a special reason for this separation.

Reply:
We name the sensors to the user "humidity", but the backend api and device itself knows them as "hum".   In the case of battery this looks fine, but..

Reply:
The key is only used for the unique id. That's not user facing. It's ok to use `"hum"` etc there.